neurons/competition_config.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
Will we store the competition config in the subnet repo or some separate service/hugging face and fetch it?

Reply:
in the repo